packages/shared/package.json

```
@@ -0,0 +1,35 @@
+{
+  "name": "@booking-services/shared",
+  "version": "0.0.0",
+  "description": "Some code that we could share with other teams",
+  "author": "Hector Esau Macias Javier <hecttoressau@gmail.com>",
+  "homepage": "https://github.com/Platzi-Master-C9/backend-services/tree/master/packages/shared#readme",
+  "license": "ISC",
+  "main": "lib/shared.js",
+  "directories": {
+    "lib": "lib",
+    "test": "__tests__"
+  },
+  "files": [
+    "lib"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Platzi-Master-C9/booking-services.git"
+  },
+  "scripts": {
+    "test": "jest"
```

Can you add the eslint commands here?
```suggestion
    "test": "jest",
    "lint": "eslint .",
    "lint": "eslint . --fix"
```